src/rock/models/likes/mutations.js

```diff
@@ -0,0 +1,4 @@
+
+export default [
+   "toggleLike(nodeId: String!): [Node]",
```

Comment:
This should be a `MutationResponse` that adds a `like: Node` which isn't an array. That way the client can easily update its client cache